core-aam/acacia/role-map.json

```diff
@@ -0,0 +1,190 @@
+rolemap = {
+  "role-map-blockquote" : {
+    "Atspi" : [
+       [
+          "property",
+          "role",
+          "is",
+          "block quote"
+       ]
+    ],
+    "AXAPI" : [
+       [
+          "property",
+          "AXRole",
+          "is",
+          "AXGroup"
+       ],
+       [
+          "property",
+          "AXSubrole",
+          "is",
+          "<nil>"
+       ]
+    ],
+    "IAccessible2" : [
+       [
+          "property",
+          "role",
+          "is",
+          "IA2_ROLE_BLOCK_QUOTE"
+       ]
+    ],
+    "MSAA" : [
+       [
+          "property",
+          "role",
+          "is",
+          "ROLE_SYSTEM_GROUPING"
+       ]
+    ],
+    "UIA" : [
+       [
+          "property",
+          "ControlType",
+          "is",
+          "Group"
+       ],
+       [
+          "property",
+          "LocalizedControlType",
+          "is",
+          "blockquote"
+       ]
+    ]
+  },
```

Comment:
These are the objects that are currently being used in the manual tests, the directory above this one: 
https://github.com/web-platform-tests/wpt/blob/master/core-aam/blockquote-manual.html

We could have an entry for every role in the role mapping tables of core-aam. Then, when a role from a different AAM points to core-aam, you can use the same map here. Currently the keys are the id refs for the mapping table, such as: https://w3c.github.io/core-aam/#role-map-blockquote